tests/view/test_slice.cpp

```diff
@@ -0,0 +1,119 @@
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+#include "Kokkos_Core.hpp"
+
+#include "kokkos-utils/view/extents.hpp"
+#include "kokkos-utils/view/slice.hpp"
+
+using execution_space = Kokkos::DefaultExecutionSpace;
+
+/**
+ * @file
+ *
+ * @addtogroup unittests
```

Comment:
We noticed that it may not necessarily work to set both `@file` and `@addtogroup`. Let's remove `@file` here and to a search replace in the repository to solve similar cases?

Reply:
I will not do this here :wink: